docker/entrypoint.sh

```diff
@@ -58,6 +58,10 @@ case $1 in
       deploy
     ;;
 
+    tests)
```

Comment:
pelase find another name for this entry, as tests is too generic and it is very likely that we will add other tests ( our own tests) too.